gui/tasks/distribution.js

```diff
@@ -11,11 +11,14 @@ const shouldNotarize = process.argv.includes('--notarize');
 const universal = process.argv.includes('--universal');
 const release = process.argv.includes('--release');
 
-const targetsIndex = process.argv.indexOf('--targets');
-let targets = null;
+const targets = getOptionValue('--targets');
+const hostTargetTriple = getOptionValue('--host-target-triple');
```

Comment:
Maybe "host target" is a bit of a contradiction? My interpretation is that the target is the machine you're building for, so the variable should just be called `hostTriple`.

Reply:
I think "target triple" is the name of [the format](https://wiki.osdev.org/Target_Triplet). And it is the target triple for the host running the script, even though it might not be a target.